tasks/icons.js

```diff
@@ -28,13 +28,21 @@ task('delete-icons', () =>
 
 task('clean-icons', () =>
   src('node_modules/@a4u/a4u-spectrum-open-source/assets/*/*.svg')
-    .pipe(replace(/<defs>[\s\S]*?<\/defs>/m, ''))
```

Comment:
Just curious why remove these two lines. As the style will be inlined or applied below L32 - L33, the content in <defs> should be useless, right?

Reply:
I used svgmin to delete the styles element 
And then the defs is an empty element which I also remove with an svgmin plugin

Reply:
And when you remove the defs in the first step, you can not inline the styles, as you deleted the style values

Reply:
And in line 32 - 33 i just handles two styles which could be transferred in to opacities.
I need too keep the defs to inline all other styles, transfer them from styles to attributes and then delete all attributes except the opacity